documentation/asciidoc/computers/processors/bcm2712.adoc

```diff
@@ -24,3 +24,20 @@ Headline features include:
 ** H264 1080p30 encode (from ISP) ~30–40% CPU
 
 In aggregate, the new features present in BCM2712 deliver a performance uplift of 2-3× over Raspberry Pi 4 for common CPU or I/O-intensive use cases.
+
+=== Vulnerabilities and mitigations
+
+The Cortex-A76 used on the BCM2712 has known vulnerabilites that are all mitigated in Raspberry Pi OS.
+
+To determine the full list of vulnerabilities and the mitigations, you can use the following on the command line which will list all those in place.
+
+```bash
+$ lscpu | grep Mitigations
```

Comment:
I guess that technically this will only list Vulnerabilities that already have a Mitigation, but might not list (future) Vulnerabilities that _don't_ yet have a Mitigation in place? So perhaps a better command would be:
```bash
$ lscpu | grep Vulnerability | grep -v "Not affected"
```
?

Either way, your original `grep` command needs to use `Mitigation` rather than `Mitigations` :wink:

Reply:
Have used your grep